README.md

```diff
@@ -230,24 +230,34 @@ returned, only the diff.
 
 Currently accepts the same options as the read stream except for reverse.
 
-#### `watcher = db.watch([range], [onchange])`
+#### `watcher = db.watch([range])`
 
 Listens to changes that are on the optional `range`.
 
 `range` options are the same as `createReadStream` except for reverse.
 
-`watcher.destroy()`\
-Closes the watcher.
+Usage example:
+```js
+for await (const { current, previous } of watcher) {
+  console.log(current.version)
+  console.log(previous.version)
+}
+```
 
-`watcher.on('change', (newVersion, oldVersion) => {})`\
-Emitted after a feed change.
+Or manually calling `next()`:
+```js
+const { done, value } = await watcher.next()
+if (done) return
 
-`watcher.on('error', onerror)`\
-Critical and unexpected errors will be thrown, but watcher is normally graceful.\
-Also, watcher auto closes on errors.
+console.log(value.current.version)
+console.log(value.previous.version)
+```
+
+Returns a new value after a feed change, `current` and `previous` are snapshots that are auto-closed before next value.
```

Comment:
Although it's already said implicitly, I'd add an additional warning 'Note that you should not close the yielded snapshots yourself', because if they do an error will be thrown (upon setting `this.previous = this.current.snapshot()` in the next iteration)